plugin.xml

```diff
@@ -132,7 +132,9 @@ to config.xml in order for the application to find previously stored files.
             </feature>
             <allow-navigation href="cdvfile:*" />
         </config-file>
-
+	<edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
+	    <application android:requestLegacyExternalStorage="true" />
+	</edit-config>
```

Comment:
```suggestion
    <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
        <application android:requestLegacyExternalStorage="true" />
    </edit-config>
```

Indentation is tabs, the rest of the file are spaces. Sorry for being nitpicky here, but can you please correct those? 🙈

Reply:
Sometimes I'm lazy and do the changes in vim instead of VS code and forget that it doesn't deal with whitespace/tabs appropriately :see_no_evil: